### DISCUSSION TITLE
Is there a way to update the a Table query inside a ViewField based on the Form fields?

### DISCUSSION THREAD

**Discussion by omar-co** [filamentphp/filament#6750] [Help]
I have a form with Tabs, in the first tab I have two selects (branch_id and department_id), in the last tab I have a ViewField with a Table inside that shows the data of the Budget model, is there a way in which they can interact with each other? 

I would like to filter the table data by the branch_id and department_id in the first tab, and then save the ids of the rows selected in the table.

**Comment by pboivin** [comment]
Hi @omar-co, it's technically possible to use a Filament Table inside of a Form (I'm doing it on one project), but it's a slippery slope. You will likely encounter weird behaviours and limitations. Basically, HTML doesn't allow nested `<form>` tags, and the Filament Table itself contains a few.

To your question, my suggestion would be to explore Bulk Actions :
https://filamentphp.com/docs/2.x/admin/resources/listing-records#bulk-actions

If your table is wrapped inside of a custom Livewire component, you can get the selected records in a custom Bulk Action and emit the IDs up to your page component/main form.

Similarly for dependent fields, you could emit from the main form to your custom component and reload the table accordingly.

Hope this can help!
